src/signals/incident-management/containers/LegacyIncidentSplitContainer/components/IncidentPart/index.test.js

```diff
@@ -14,11 +14,6 @@ jest.mock('containers/App/selectors', () => ({
   ...jest.requireActual('containers/App/selectors'),
 }));
 
-const subCategories = categoriesFixture.results
-  .filter(modelSelectors.filterForSub)
-  // mapping subcategories to prevent a warning about non-unique keys rendered by the SelectInput element 🙄
-  .map(subCat => ({ ...subCat, key: subCat._links.self.href }));
-
```

Comment:
Is this a change that is required for changing the subcategory in the incident form?

Reply:
no.

Reply:
no